airbyte-ci/connectors/pipelines/pipelines/actions/environments.py

```diff
@@ -678,30 +623,47 @@ def with_integration_base(context: PipelineContext, build_platform: Platform) ->
     )
 
 
-def with_integration_base_java(context: PipelineContext, build_platform: Platform, jdk_version: str = "17.0.4") -> Container:
+def with_integration_base_java(context: PipelineContext, build_platform: Platform) -> Container:
     integration_base = with_integration_base(context, build_platform)
+    yum_packages_to_install = [
+        "tar",  # required to untar java connector binary distributions.
+        "openssl",  # required because we need to ssh and scp sometimes.
+        "findutils",  # required for xargs, which is shipped as part of findutils.
+    ]
     return (
         context.dagger_client.container(platform=build_platform)
-        .from_(f"amazoncorretto:{jdk_version}")
-        .with_directory("/airbyte", integration_base.directory("/airbyte"))
+        # Use a linux+jdk base image with long-term support, such as amazoncorretto.
+        .from_(f"amazoncorretto:{AMAZONCORRETTO_TAG}")
```

Comment:
nit: any reason to not store the full image name in a constant like:
`AMAZONCORETTO_IMAGE = f`amazoncoretto:{AMAZONCORETTO_TAG}`

Reply:
No, indeed. Will do.

Reply:
Done.